ilp-core/src/test/java/org/interledger/core/InterledgerAddressTest.java

```diff
@@ -196,6 +196,16 @@ public void testStartsWithInterledgerAddress() {
     assertThat(address.startsWith(InterledgerAddress.of("test1.foo.bob")), is(false));
   }
 
+  @Test
+  public void testStartsWithInterledgerAddressPrefix() {
+    final InterledgerAddress address = InterledgerAddress.of("g.foo.bob");
+    assertThat(address.startsWith(InterledgerAddressPrefix.of("g")), is(true));
+    assertThat(address.startsWith(InterledgerAddressPrefix.of("g.foo")), is(true));
```

Comment:
I also create #212 for later tracking - I don't think we should implement that in this PR, but might be a good fast-follower once this is solid.